src/common/comp/knowhere_config.cc

```diff
@@ -147,5 +149,11 @@ KnowhereConfig::FreeGPUResource() {
     knowhere::GPUResMgr::GetInstance().Free();
 #endif
 }
+void
+KnowhereConfig::SetRaftMemPool(size_t init_size, size_t max_size) {
+#ifdef KNOWHERE_WITH_RAFT
+    knowhere::SetRaftMemPool(init_size, max_size);
```

Comment:
Can we not compile `KnowhereConfig::SetRaftMemPool` if RAFT is not enable instead of just skipping calling `knowhere::SetRaftMemPool`.